internal/orchestrion/_integration/internal/trace/span.go

```diff
@@ -159,3 +160,137 @@ func printableSpanAttribute(attrs map[string]any, key string) string {
 	}
 	return fmt.Sprintf("%+v", val)
 }
+
+// FromSimplified parses a trace tree from a simplified string format, with the following criteria:
+// - 1 span per line
+// - Span format is [span.name | resource.name | component | span.kind]
+// - All values are optional except span.name
+// - Indentation (4 spaces or 1 tab) is used to represent parent-child relationships.
+//
+// Example:
+//
+//	[http.request | GET / | net/http | client]
+//	    [http.request | GET / | net/http | server]
+//	        [echo.someMiddleware | echo-ctx-someMiddleware]
+//	            [http.request | GET / | labstack/echo.v4 | server]
+//	                [some.span]
+//	                [other.span]
+//	[another.root]
+//	    [another.root.child1]
+//	    [another.root.child2]
```

Comment:
The one thing that frustrates me a little with the formalism is that on the call side, it's not clear what each part is... You have to read this doc... This is probably not a big deal (after all, it's still pretty expressive, and it's consistent, too...).